src/main/java/ru/itis/services/UserService.java

```diff
@@ -67,11 +67,11 @@ public TokenDto login(UserDto userDto) {
 
     public Optional<User> findUserByToken(String token) {
         String userLogin = tokensRepository.findUsernameByValue(token);
-        return usersRepository.findByLogin(userLogin);
+        return userRepository.findByLogin(userLogin);
     }
 
     public List<UserDto> getUsersByLogin(String login) {
-        return usersRepository.findAllByLoginIsContaining(login);
+        return userRepository.findAllByLoginIsContaining(login);
```

Comment:
Why do you need this class if it only delegates?

Reply:
some actions can not be made in controller->need of service